lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -26,7 +26,7 @@ use OpenQA::Schema::JobGroupDefaults;
 use Class::Method::Modifiers;
 use OpenQA::Utils qw(log_debug parse_tags_from_comments);
 use Date::Format 'time2str';
-use OpenQA::JobTemplates 'dump_yaml';
+use OpenQA::YAML 'dump_yaml';
```

Comment:
If the module is called yaml then you could rename the method to just 'dump'. And 'load' accordingly

Reply:
We should only be aware that `dump` is also the name of the builtin function that core dumps, which will be called if one forgets to import.
Of course that should be catched by our testsuite, just saying.

Reply:
We import it by name... unless you're implying we change that I don't think `dump` would be a practical name for the method. Same for `load`.

Reply:
The names are fine. Don't shorten them which will only make name conflicts more likely and code which calls them harder to read.